.github/workflows/ci.yml

```
@@ -0,0 +1,90 @@
+          STT_BASE_URL: "http://localhost:8000/v1"
+          TTS_API_KEY: ${{ secrets.TTS_API_KEY || 'dummy-key' }}
+          STT_API_KEY: ${{ secrets.STT_API_KEY || 'dummy-key' }}
+        run: |
+          # Execute tests that might interact with the external Speaches container
+          # Currently, this runs all tests, but as the suite grows, you may
+          # restrict this to a specific directory or use pytest markers (e.g., -m integration)
+          python -m pytest tests/test_audio_service.py -v
+
+      - name: Fix cache permissions
+        if: always()
+        run: sudo chown -R $USER models
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:15: Double quote to prevent globbing and word splitting

(shellcheck)